frontend/src/app/Catalog/CatalogView/CatalogView.tsx

```diff
@@ -130,11 +256,115 @@ const CatalogView = (props: CatalogViewProps) => {
 		});
 	};
 
-	const enrollPath = legacyId
-		? `/enrollment/0-${legacyId}-${semester.replace(' ', '-')}-all`
-		: `/enrollment`;
+	
 
-	const gradePath = legacyId ? `/grades/0-${legacyId}-all-all` : `/grades`;
+
+	const [hoveredClass, setHoveredClass] = useState<any>(false);
```

Comment:
Perhaps we can work on the grades chart itself to handle this sort of hovering functionality. Not sure